src/sessionTypes/cliAuth.js

```diff
@@ -0,0 +1,153 @@
+import auth0 from 'auth0-js';
+import axios from 'axios';
+
+/**
+ * A SessionType that allows the user to authenticate with Auth0 and
+ * then gain a valid JWT from the Contxt Auth service. This would only
+ * be used in command line applications such as `contxt-cli`.
+ *
+ * @type SessionType
+ *
+ * @typicalname contxtSdk.auth
+ *
+ * @example
+ * const ContxtSdk = require('@ndustrial/contxt-sdk');
+ *
+ * const contxtService = new ContxtSdk({
+ *   config: {
+ *     auth: {
+ *       clientId: 'bleED0RUwb7CJ9j7D48tqSiSZRZn29AV'
+ *     }
+ *   },
+ *   sessionType: 'cliAuth'
+ * });
+ */
+
+class CliAuth {
```

Comment:
You need a couple more methods here:
1. `getCurrentApiToken`
2. `isAuthenticated`

These are both [documented](https://github.com/ndustrialio/contxt-sdk-js/blob/276a9bbafbca78a5e8976019c7088bf9b69c1308/docs/Typedefs.md#sessiontype--object]) as methods that are required for a SessionType